joomla/joomla-cms
# [RFC] Joomla Dialog, implementation for Media, User Fields

Pull Request for Issue # .

### Summary of Changes

- uses the Joomla Dialog from #40150 for the fields User and Media

### Testing Instructions

- Apply the patch
- Run `npm ci`
- Check that intro, full text images and author fields are working as before

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request

User Field
![Screenshot 2023-03-20 at 11 33 33](https://user-images.githubusercontent.com/3889375/226314894-58bb3a57-76c3-401a-ae7d-e38a3cf6069e.png)

Media Field
![Screenshot 2023-03-20 at 11 33 48](https://user-images.githubusercontent.com/3889375/226314983-38f7989b-5db7-49cb-b787-abe043861b58.png)


### Link to documentations
Please select:
